app/pages/collections/list.cjsx

```diff
@@ -64,24 +132,40 @@ List = React.createClass
             </div>
             <nav>
               {if meta
+                buttonClasses = classNames {
+                  "pill-button": true
+                  "project-pill-button": @props.project?
+                }
                 <nav className="pagination">
                   {for page in [1..meta.page_count]
                     active = (page is +location.query.page) or (page is 1 and not location.search)
                     <Link
-                      key={page}
-                      to={"#{@props.location.pathname}?page=#{page}"}
-                      activeClassName="active"
-                      className="pill-button"
-                      style={border: "2px solid" if active}>
+                      key = {page}
```

Comment:
Yeah, I suppose it does, but everywhere else in the app and general HTML conventions are unspaced.

I suspect that this line is the culprit: `style = {border: "2px solid" if active}>`

The trailing `>` should be removed.

`style={ {border: "2px solid"} if active}` may also fix the syntax parsing.

@srallen, what's your editor?  Do either of those suggestions work for you?